api/requirements.txt

```diff
@@ -0,0 +1,104 @@
+oauthlib==3.2.2 ; python_version >= "3.8" and python_version < "3.11"
+opt-einsum==3.3.0 ; python_version >= "3.8" and python_version < "3.11"
+packaging==23.0 ; python_version >= "3.8" and python_version < "3.11"
+pandas==1.5.3 ; python_version >= "3.8" and python_version < "3.11"
+pathy==0.10.1 ; python_version >= "3.8" and python_version < "3.11"
+pillow==9.4.0 ; python_version >= "3.8" and python_version < "3.11"
+pluggy==1.0.0 ; python_version >= "3.8" and python_version < "3.11"
+preshed==3.0.8 ; python_version >= "3.8" and python_version < "3.11"
+protobuf==4.22.1 ; python_version >= "3.8" and python_version < "3.11"
+pyasn1-modules==0.2.8 ; python_version >= "3.8" and python_version < "3.11"
+pyasn1==0.4.8 ; python_version >= "3.8" and python_version < "3.11"
+pydantic==1.10.7 ; python_version >= "3.8" and python_version < "3.11"
+pyparsing==3.0.9 ; python_version >= "3.8" and python_version < "3.11"
+pytest==7.2.2 ; python_version >= "3.8" and python_version < "3.11"
+python-dateutil==2.8.2 ; python_version >= "3.8" and python_version < "3.11"
+pytz==2023.2 ; python_version >= "3.8" and python_version < "3.11"
+pyyaml==6.0 ; python_version >= "3.8" and python_version < "3.11"
+regex==2023.3.23 ; python_version >= "3.8" and python_version < "3.11"
+requests-oauthlib==1.3.1 ; python_version >= "3.8" and python_version < "3.11"
+requests==2.25.1 ; python_version >= "3.8" and python_version < "3.11"
+rfc3986[idna2008]==1.5.0 ; python_version >= "3.8" and python_version < "3.11"
+rsa==4.9 ; python_version >= "3.8" and python_version < "3.11"
+scikit-learn==1.0.2 ; python_version >= "3.8" and python_version < "3.11"
+scipy==1.10.1 ; python_version >= "3.8" and python_version < "3.11"
+setuptools-scm==7.1.0 ; python_version >= "3.8" and python_version < "3.11"
+setuptools==67.6.1 ; python_version >= "3.8" and python_version < "3.11"
+six==1.16.0 ; python_version >= "3.8" and python_version < "3.11"
+smart-open==6.3.0 ; python_version >= "3.8" and python_version < "3.11"
+sniffio==1.3.0 ; python_version >= "3.8" and python_version < "3.11"
+spacy-legacy==3.0.12 ; python_version >= "3.8" and python_version < "3.11"
+spacy-loggers==1.0.4 ; python_version >= "3.8" and python_version < "3.11"
+spacy==3.5.1 ; python_version >= "3.8" and python_version < "3.11"
+srsly==2.4.6 ; python_version >= "3.8" and python_version < "3.11"
+starlette==0.26.1 ; python_version >= "3.8" and python_version < "3.11"
+tensorboard-data-server==0.7.0 ; python_version >= "3.8" and python_version < "3.11"
+tensorboard-plugin-wit==1.8.1 ; python_version >= "3.8" and python_version < "3.11"
+tensorboard==2.12.0 ; python_version >= "3.8" and python_version < "3.11"
+tensorflow-estimator==2.12.0 ; python_version >= "3.8" and python_version < "3.11"
+tensorflow-io-gcs-filesystem==0.31.0 ; python_version >= "3.8" and python_version < "3.11" and platform_machine != "arm64" or python_version >= "3.8" and python_version < "3.11" and platform_system != "Darwin"
```

Comment:
Wow, I don't even know what Darwin is. Same question, is this portable?

Reply:
Tested installing the requirements.txt file in CDU linux server, which went ok.